eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -2147,5 +2147,12 @@
     <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
   </Match>
 
+  <!-- Return poll status as failed instead of throwing any exception -->
```

Comment:
Is there a particular reason we need to update both this file and the spotbugs file specific to storage? I'd expect we would only need to update one of them especially because this one does not have all of the storage specific excludes in it.